jbig2/jbig2_compress_image.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * This example showcases the conversion of the jpg encoded image
+ * into jbig2 encoding format.
+ *
+ * The result jbig2 file is compressed with a lossless method stored with
+ * the standard  jbig2 - .jb2 extension. The file compressed in this example
+ * is stored in just 377 bytes - compared to the 142 794 bytes for original image.
+ * This gives compression ratio (uncompressed size/compressed size) of 378.76, which leads to 99.735%
+ * space savings for given example.
+ */
+
+package main
+
+import (
+	"fmt"
+	"image"
+	// load jpeg decoder
+	_ "image/jpeg"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/core"
+)
+
+func main() {
+	// let's read an jpeg rgba image from the file
```

Comment:
Please start comments with capital letter and end them in `.` throughout the example, for consistency.

Reply:
Done.